server/src/tenant/metadata/data-source/data-source.service.ts

```diff
@@ -48,6 +58,40 @@ export class DataSourceService implements OnModuleInit, OnModuleDestroy {
     );
   }
 
+  private async createMigrationTable(
+    queryRunner: QueryRunner,
+    schemaName: string,
+  ) {
+    await queryRunner.createTable(
+      new Table({
```

Comment:
Note: Ideally, this should be based on server/src/tenant/metadata/migration-generator/tenant-migration.entity.ts but I couldn't make it work